client/src/components/ThemeToggle/index.js

```diff
@@ -2,11 +2,15 @@ import React, { useContext } from 'react';
 import { ThemeContext } from '../../ThemeContext.js';
 import { FiSun, FiMoon } from 'react-icons/fi';
 import styles from './Style.module.css';
+import storage from '../../utils/storage';
 
 const ThemeToggle = () => {
   const [darkMode, setDarkMode] = useContext(ThemeContext);
 
+  setDarkMode(storage.get('theme'));
```

Comment:
State should not be set while component renders . You can do it in `useEffect` hook with empty dependencies. It behaves similar to `componentDidMount`

Reply:
![Screenshot from 2020-08-15 15-36-09](https://user-images.githubusercontent.com/62807661/90310203-1c43dd80-df0d-11ea-9c2a-bc3c86dc3571.png)

I am getting this error now. Not sure what the reason is.